src/dotnet/APIView/APIViewWeb/Startup.cs

```diff
@@ -180,6 +181,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IAuthorizationHandler, RevisionOwnerRequirementHandler>();
             services.AddSingleton<IAuthorizationHandler, ApproverRequirementHandler>();
             services.AddSingleton<IAuthorizationHandler, AutoReviewModifierRequirementHandler>();
+            services.AddHostedService<ReviewBackgroundHostedService>();
```

Comment:
Given you are using the DI container system already, and you are doing logging in the review service, then it might make sense to pick up the logging extensions infrastructure and pass in an ```ILogger<ReviewBackgroundHostedService>``` to that service. It will still publish telemetry to Application Insights, but also publish output to the console for local debugging etc.

Reply:
I will revisit this part and make changes accordingly. Based on the discussion with Pavel, logging service won't be able to group insights. In any case, I will explore this option more and make changes in different PR if required. Thanks